tutorials/Consistent_Scores.ipynb

```diff
@@ -0,0 +1,465 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Consistent scores\n",
+    "\n",
+    "A \"consistent scoring function\" is a scoring function where a forecaster's expected score will be optimised by following a forecast directive. An example of a forecast directive is to \"predict the mean (i.e., expected) rainfall amount\". Using a consistent scoring function ensures that a forecaster or forecast system developer isn't faced with the dilemma of whether to \n",
+    "\n",
+    "1. produce forecasts (or develop a forecast system) that optimises the expected verification score, or\n",
+    "2. produce an honest forecast (or develop an honest forecast system) that is consistent with the forecast directive. \n",
+    "\n",
+    "Using a consistent scoring function to evaluate forecast performance ensures that objectives both 1 and 2 can be met without any tensions. For example, the Mean Squared Error (MSE) is a consistent score for forecasting the mean (i.e., expected) rainfall amount, since forecasting the \"mean rainfall amount\" will minimise the expected MSE. However the Mean Absolute Error (MAE) is not consistent with forecasting the mean value and is instead consistent with forecasting the median value. Consistent scoring functions are formally defined in ([Gneiting et al., 2011a](https://www.tandfonline.com/doi/abs/10.1198/jasa.2011.r10138)).\n",
+    "\n",
+    "\n",
+    "You may already know that the MSE is a consistent score for forecasting the mean and the MAE is a consistent score for forecasting the median. But did you know that there are a whole family of scores that are consistent for prediciting the mean, median, or quantiles? \n",
+    "\n",
+    "The consistent scoring module in `scores` provides access to these families of consistent scores hat can be used to tailor emphasis of predictive performance based on decision thresholds of interest.\n",
+    "\n",
+    "### Example 1. A score consistent with the mean\n",
+    "Let's jump into an example that illustrates a scoring function that is consistent with predicting the mean value using `scores`. Suppose we want to evaluate the performance of expected daily precipitation forecasts, but we want to place increasing importance on correctly forecasting extreme values. \n",
```

Comment:
If that's all that's wanted, what's wrong with MSE, or even mean cubed error? (not saying there's no answer, but that is probably what will jump into people's heads)

Reply:
Maybe a comment like "MSE could potentially be used, but the approach below can be extended to allow for expectiles other than the mean, skewed weightings or to take into account specific ranges."

Reply:
MSE is consistent, but I don't think cubed error is.

MSE weights all decision thresholds equally and doesn't meet the requirement to "place increasing importance on correctly forecasting extreme values". MSE is discussed further along in the tutorial.

The point isn't that we want to use scores that are consistent for different expectiles since only scores with alpha=0.5 will be consistent in this case. The point is that we want to be able to weight the importance of different decision thresholds.

I added a sentence, "MSE is consistent with predicting the mean value, but weights all decision thresholds equally (as discussed further on)."